<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro_Produtos</class>
 <widget class="QFrame" name="Cadastro_Produtos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="txtTitulo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CADASTRO DE PRODUTOS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtCodigo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CÓDIGO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtDescricao">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DESCRICAO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtPreco">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PREÇO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtCategoria">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CATEGORIA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCodigo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineDescricao">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePreco">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnviar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ENVIAR</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioInformatica">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>INFORAMATICA</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Categoria</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioAlimentos">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ALIMENTOS</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Categoria</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioEletronicos">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ELETRÔNICOS</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Categoria</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnListar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LISTAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Categoria"/>
 </buttongroups>
</ui>
